FCMS/views/default.py

```diff
@@ -6,7 +6,7 @@
 from .. import models
 
 
-@view_config(route_name='home', renderer='../templates/comingsoon.jinja2')
+@view_config(route_name='home', renderer='../templates/landing.jinja2')
```

Comment:
Undo, the route change will be done by creating a new view, not editing the scaffold view.